integrations/azure/README.md

```diff
@@ -0,0 +1,44 @@
+# Azure ORT starter pipeline
+
+The starter pipeline allows you to easily run an ORT scan on Azure DevOps.
+
+To run the pipeline, simply:
+
+- create a new YAML-based pipeline in your Azure DevOps project based on the pipeline file in this repository
+- run the pipeline with the preferred set of parameters
+
+## Using credentials for my repository
+
+The pipeline supports using simply credentials (username, password) to download a repository.
+Credentials can be set up in the following way:
+
+- Check the parameter `Use repository credentials as defined in variable group ort_repo_credentials`
+- Add a variable group called `ort_repo_credentials` (via "Library" -> "Variable groups")
+- Add the variables `USERNAME` and `PASSWORD` to this group (for security, make password a secret)
+- NB: The names have to be exactly the ones described above: Azure devops doesn't currently allow dynamic variable groups
+
+This will result in the build using these credentials to download your project
+
+## Using a different repository for ORT configuration
+
+The pipeline supports using a different repository (with different credentials) to use for a pipeline run.
+Simply fill the two parameters `Config Version Control URL` and optionally change `Config Revision`
+If you need authentication to access the repository, you can do so via login and password:
+
+- Check the parameter `Use credentials for ort as defined in variable group ort_config_credentials`
+- Add a variable group called `ort_config_credentials` (via "Library" -> "Variable groups")
+- Add the variables `ORT_USERNAME` and `ORT_PASSWORD` to this group (for security, make password a secret)
+- NB: The names have to be exactly the ones described above: Azure devops doesn't currently allow dynamic variable groups
```

Comment:
NB:? What do you mean here?

Reply:
I think "Nota Bene". We can put "Note:" too.